src/compat/browser_detection.ts

```diff
@@ -31,10 +31,14 @@ const isSamsungBrowser : boolean = /SamsungBrowser/.test(navigator.userAgent);
 
 const isSafari : boolean = /Safari/i.test(navigator.userAgent);
 
+const isSafariMobile : boolean = typeof navigator.platform === "string" &&
+                                 /iPad|iPhone|iPod/.test(navigator.platform);
```

Comment:
Are we sure we want to put them all on the same basket ?
iPad Safari now supports MSE, all may react differently that iPhone/iPad Safari.

Reply:
I'll check on an iPad

Reply:
Same problem on an iPad with directfile contents, I could not test MSE on that device though.

This PR now only impacts iPhone/iPad/iPod in directfile mode.